.github/workflows/build.yml

```diff
@@ -1,111 +1,94 @@
+name: Build
```

Comment:
Yup, it shows the name you set in their Actions page instead of the workflow file path.
Basically, "Build" instead of ".github/workflows/build.yml"

Just cleaner in my opinion.